goat.sh

```diff
@@ -151,6 +164,13 @@ configure() {
             remove_shortcut "$3" || exit_status=1
         elif [ "$2" = "help" ] && [ "$3" = "me" ]; then
             show_help
+        elif [ "$2" = "edit" ] && [ "$3" = "shortcuts" ]; then
+            if [ -z "$EDITOR_COMMAND" ]; then
+                errcho "ERROR: goat doesn't know what command to use. Please set EDITOR."
+                exit_status=1
+            else
+                $EDITOR_COMMAND "$SHORTCUTS_FILE" < `tty` > `tty` || exit_status=1
```

Comment:
This is magic! Thanks a lot. Although I'd refactor it to `$()` instead of backticks